dependencies/bionic/dynflow/control

```diff
@@ -11,7 +11,7 @@ XS-Ruby-Versions: all
 Package: ruby-dynflow
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
-Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, ruby-algebrick (>= 0.7.0), ruby-algebrick (<< 0.8.0), ruby-apipie-params, ruby-concurrent (>= 1.0.0), ruby-concurrent (<< 2.0.0), ruby-multi-json, ruby-sequel (>= 4.0.0)
+Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, ruby-algebrick (>= 0.7.0), ruby-algebrick (<< 0.8.0), ruby-apipie-params, ruby-concurrent (>= 1.1.3), ruby-concurrent (<< 2.0.0), ruby-multi-json, ruby-sequel (>= 4.0.0)
```

Comment:
@mmoll we updated dependency on concurrent-ruby in dynflow. There should be also bump on `concurrent-ruby-edge ~> 0.4.1` but I don't see in anywhere.

Reply:
That's because ruby-concurrent 1.0.x is coming from the distro repos and contains `-edge` in that package, also.

We need to DEB package ruby-concurrent 1.1 and `-edge` like we had it in the beginning.

I'll have a look if I can handle this in the current distro package and also submit this upstream, if possible.